### ISSUE TITLE
i made a reimplement with tensorflow2

### ISSUE THREAD

**Issue by YuhangJi** [shekkizh/FCN.tensorflow#127]
i made a reimplement with tensorflow2, but it is not very well. In MIT dataset, the model loss value just decline to 2.3.

**Comment by varungupta31** [comment]
Hey @YuhangJi,
I'm tring to train the FCN on my custom dataset, in which I have only 2 classes (background + object).
Can you guide me with how to manipulate the code to achieve this?

TIA

**Comment by YuhangJi** [comment]
> i made a reimplement with tensorflow2, but it is not very well. In MIT dataset, the model loss value just decline to 2.3.

This issue is caused by the absence of a softmax function when training a model.

**Comment by YuhangJi** [comment]
> Hey @YuhangJi, I'm tring to train the FCN on my custom dataset, in which I have only 2 classes (background + object). Can you guide me with how to manipulate the code to achieve this?
> 
> TIA

No problem.
@[varungupta31](https://github.com/varungupta31)
